src/test/java/de/unistuttgart/iste/gits/content_service/api/mutation/MutationRemoveTagFromContentTest.java

```diff
@@ -7,26 +7,33 @@
 import de.unistuttgart.iste.gits.content_service.persistence.dao.ContentEntity;
 import de.unistuttgart.iste.gits.content_service.persistence.dao.TagEntity;
 import de.unistuttgart.iste.gits.content_service.persistence.repository.ContentRepository;
+import de.unistuttgart.iste.gits.content_service.persistence.repository.TagRepository;
+import de.unistuttgart.iste.gits.content_service.service.TagService;
 import jakarta.transaction.Transactional;
 import org.junit.jupiter.api.Test;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.graphql.test.tester.GraphQlTester;
 import org.springframework.test.annotation.Commit;
 
+import java.util.List;
 import java.util.Set;
 import java.util.UUID;
 
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.containsInAnyOrder;
 import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.is;
 
 @GraphQlApiTest
 @TablesToDelete({"content_tags", "content", "tag"})
 class MutationRemoveTagFromContentTest {
 
     @Autowired
     private ContentRepository contentRepository;
-
+    @Autowired
+    private TagRepository tagRepository;
+    @Autowired
+    private TagService tagServiceMock; // Mock the TagService
```

Comment:
where do you need the tag service? I dont see any usages

Reply:
> where do you need the tag service? I dont see any usages

its from my previous code, it has no use. I will remove this